src/components/bds-switch/test/bds-switch.e2e.ts

```diff
@@ -0,0 +1,43 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('switch e2e tests', () => {
+  it('should must disable', async () => {
```

Comment:
it's a good practice to keep a single responsibility to each test, in this case, you must create a single test to test only the disabled scenario and another test to the enabled scenario, instead of doing all the tests in a single function.